SYCL/ESIMD/api/functional/value.hpp

```diff
@@ -86,6 +106,8 @@ template <typename DataT> struct value {
   }
 
   static DataT denorm_min() {
+    assert(type_traits::is_sycl_floating_point_v<DataT> &&
+           "Only floating point type can be passed into this function.");
```

Comment:
please add comments for the bit values below (what half value they represent)

Reply:
I've applied this comment in this commit: 00693ecea35d0a2935c72bb247a7a9835f965a42